client/src/main/java/model/ShipMapper.java

```diff
@@ -1,18 +1,17 @@
 package model;
 
-import java.util.ArrayList;
 import java.util.List;
 
 /**
  * It maps ships to their model.
  *
- * @author Patrycja Mikulska
+ * @author Emilia Ciastek
```

Comment:
Why do you need an author tag?

Reply:
I really don't know :/ This is bad practice?

Reply:
I would not recommend to use it.